docs/superpowers/specs/2026-06-01-orchestration-mode-design.html

```
@@ -0,0 +1,299 @@
+<table>
+  <tr><th>Mechanism</th><th>Doc (next-opus)</th><th>Runs-today (0.83.0)</th><th>Forward seam</th></tr>
+  <tr><td>Effort level</td><td><code>effort:"xhigh"</code></td><td><code>output_config={"effort":"xhigh"}</code> — raw dict, per tournament.py precedent</td><td><code>ModeConfig.effort</code> constant</td></tr>
+  <tr><td>Mode reminder</td><td>mid-convo <code>role:"system"</code></td><td><code>&lt;system-reminder&gt;…&lt;/system-reminder&gt;</code> folded into the user turn's content (alternation-safe)</td><td><code>ModeConfig.reminder_transport</code> → flip to <code>SYSTEM_ROLE</code> (typed in newer SDK)</td></tr>
+  <tr><td>Standing consent</td><td>tool description text</td><td>identical — verbatim in <code>WORKFLOW_TOOL["description"]</code></td><td>none</td></tr>
+  <tr><td>Thinking</td><td><code>{"type":"adaptive"}</code></td><td>identical — valid in 0.83.0</td><td>none</td></tr>
+</table>
+<p>The top-level <code>system</code> field never changes across turns, so the cached prefix stays intact — the entire point of the design. Reminders ride in the <code>messages</code> array, placed <em>after</em> the user turn they apply to.</p>
+
+<h2 id="loop">6. The loop (semantics)</h2>
+<p>Faithful to the reference, adapted to async. Pseudocode for <code>ModeAgent.turn</code>:</p>
+<pre><code>async def turn(self, user_input: str) -> str:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)